src/views/Viewer.vue

```diff
@@ -993,6 +1071,10 @@ export default {
 		cursor: pointer;
 	}
 
+	:deep(.modal-header) {
+		background-color: rgba(0, 0, 0, 0.2);
```

Comment:
why is that? :)

Reply:
This gives the semi-transparent background for the header. Without this the file name and action icons would be barely visible on pictures with very light background.
![obraz](https://user-images.githubusercontent.com/8277636/228347478-e2266583-189f-4180-b7ea-7a46fe7d52f1.png)

I have tried to make it similar in appearance to an equivalent bar in the Android app - see screenshot below:
![obraz](https://user-images.githubusercontent.com/8277636/228347846-8d31cae3-5ea6-4c34-a4d3-fc38cd16c158.png)

Reply:
Unfortunately that would not work out with bright theme viewer mode (not related to the global nextcloud theming), can you try this with he Text app for example? They use Viewer with a bright theme

Reply:
I have tried with the Text app and it is readable. One could even argue that having different background for the header is a good thing. :wink: Obviously this should be verified by the design team.

![obraz](https://user-images.githubusercontent.com/8277636/228663316-53c4ff6c-6551-43a7-b830-884fafc2c848.png)

But - there is another problem that is much more serious. 

In full screen mode content "slides" underneath the header - to use as much screen height as possible. This works good when browsing photos, but does not work at all for the text editor. Here is how it looks on Android.

![obraz](https://user-images.githubusercontent.com/8277636/228664192-f4cfbaf9-641b-4263-af0e-f4a502f92f88.png)

The header is not visible at all. :disappointed:

I guess we could have two modes of entering full screen:

- **aggressive** - used probably only for images when we "slide" content under the header
- **normal** - used for everything else, that would keep content below the header

Then the text editor would look like this in full screen:
![obraz](https://user-images.githubusercontent.com/8277636/228665344-da9c0706-ed4c-41f7-a012-ce3a45b4c38c.png)

I can implement these two modes in viewer, but I am not sure how to control which mode is used when. Could you please suggest something?

Reply:
cc @nextcloud/office

Reply:
I'd say full-screen is only needed for images (and possibly videos).

Text files don't need fullscreen, and also should not have their header colored differently. :) So it's fine if they stay as they are.

Reply:
@skjnldsv - In https://github.com/nextcloud/viewer/pull/1588/commits/0ec354364a02260d22000dddad42ebb7f81b8cfe I have implemented the two modes of entering full screen that I had mentioned earlier. These are controlled by the MIME type of the displayed file. I have reused heuristic from **canEdit**.

Please have a look.

Reply:
@jancborchardt - text files in full screen mode now look like this:
![obraz](https://user-images.githubusercontent.com/8277636/232345380-870f5414-4ff0-413f-9e5f-033abd5fbd1c.png)